aten/src/ATen/record_function.h

```diff
@@ -126,11 +126,36 @@ struct TORCH_API RecordFunction {
     return state_->sequence_nr_;
   }
 
-  const std::vector<c10::IValue>& inputs() const {
+  inline const std::vector<c10::IValue>& inputs() const {
```

Comment:
nit: putting `inline`  on a function that's already in the class body does nothing

Reply:
Yes. @ilia-cher, if you don't mind, I'll remove the other `inline` for all the member functions in this header to be consistent.